### PR TITLE
[CM-1326] Added option to rate conversation anytime from the menu

### PR DESCRIPTION
**Pull request by pahadi777**
Added option to rate the conversation anytime from the conversation menu which can be enabled/disabled anytime from the `applozic-settings.json` file

### CODE REVIEW

File: app/src/main/assets/applozic-settings.json [hunk 1]
```diff
@@ -163,5 +163,6 @@
   },
   "receiverNameTextColor": "#5C6677",
   "innerTimestampDesign": false,
-  "oneTimeRating": true
+  "oneTimeRating": true,
+  "ratingFromConversationEnabled" : true
```
Comment (amntoppoKM) [inline app/src/main/assets/applozic-settings.json:167]: name this to "rateConversationMenuOption"

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java [hunk 2]
```diff
@@ -1679,6 +1684,7 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         menu.findItem(R.id.unmuteGroup).setVisible(false);
         menu.findItem(R.id.deleteConversation).setVisible(false);
         menu.findItem(R.id.logout).setVisible(alCustomizationSettings.isLogoutOptionFromConversation());
+        menu.findItem(R.id.rate_conversation).setVisible(alCustomizationSettings.isRatingFromConversationEnabled());
```
Comment (amntoppoKM) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java:1687]: We have one more option to enable/disable CSAT. It is available in dashboard. We are storing it in App Setting Preference.
use this check here as well: `themeHelper.isCollectFeedback()`

**Review by amntoppoKM — CHANGES_REQUESTED** [review]

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java (hunk 2 above)
Comment (pahadi777) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java]: Done as requested

**Review by amntoppoKM — APPROVED** [review]

**Comment by Sathyan-KM** [comment]
@pahadi777 resolve the conflicts

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/AlCustomizationSettings.java [hunk 3]
```diff
@@ -136,9 +136,13 @@ public class AlCustomizationSettings extends JsonMarker {
...
     private boolean oneTimeRating;
+    private boolean restartConversationButtonVisibility = true;
+    private boolean rateConversationMenuOption;
     private boolean javaScriptEnabled;
-    private boolean restartConversationButtonVisibility;
-    
+
+    public boolean isRateConversationMenuOption() {
+        return rateConversationMenuOption;
+
```
Comment (Sathyan-KM) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/AlCustomizationSettings.java:145]: this is creating build issue @pahadi777

**Review by Sathyan-KM — CHANGES_REQUESTED** [review]

**Review by Sathyan-KM — APPROVED** [review]

**Pull request merged**